docs/asdf/release_and_support.rst

```diff
@@ -0,0 +1,102 @@
+.. currentmodule:: asdf
+
+.. _release_and_support:
+
+*************************************************
+Release Cycle and Major Dependency Support Policy
+*************************************************
+
+This document describes the general plan for releases of the ASDF library, along
+with how long we intend to support each release. Moreover, it also describes our
+policy for how long we support versions of CPython and NumPy. Note that this is
+a living document, and may be updated at any time.
+
+.. _release_cycle:
+
+Release Cycle
+=============
+
+ASDF is and will continue to be developed on a rolling release cycle, as ASDF is
+still under active development. This means that ASDF will not have a fixed release
+schedule, but rather will be released as needed.
```

Comment:
```suggestion
As ASDF is still under active development, it will continue to be developed on a 
rolling release cycle.  This means that ASDF will not have a fixed release 
schedule, but rather will be released as needed.
```